src/main/java/csv/PersonCSV.java

```diff
@@ -0,0 +1,32 @@
+package main.java.csv;
+
+import java.io.Serializable;
+
+public class PersonCSV implements Serializable {
```

Comment:
This class fully duplicate the Person class:
https://github.com/Buga1chukY/JavaPractice/blob/master/src/main/java/serialization/Person.java

Reply:
Problem still present